docs/maintainer/infrastructure.md

```diff
@@ -33,42 +83,84 @@ Smithy contains maintenance code for conda-forge, which is used by the `conda-sm
+Smithy can be used beyond conda-forge's purposes. For example, it can be used to set up self-hosted Azure agents for non-conda-forge infrastructures.
 (You could also consider using [Azure virtual machine scale set agents](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/scale-set-agents?view=azure-devops),
 which could be less expensive to run than permanently active agents.)
 
-<a id="web-services"></a>
-
 ### Web services
 
 The Heroku app providing the conda-forge web services lives in [conda-forge/conda-forge-webservices](https://github.com/conda-forge/conda-forge-webservices).
 Please note that the code logic provided by the app is in the `Smithy` repository.
 
 Bugs or suggestions regarding the service functionality should therefore be opened in `conda-forge/conda-smithy`'s [bug tracker](https://github.com/conda-forge/conda-smithy/issues).
 
-<a id="conda-forge-pinning"></a>
-
 ### conda-forge pinning
 
+Hosts the global pinnings for conda-forge, and the ongoing migrations.
+
+- ⚙️ Deployed in [Anaconda.org](https://anaconda.org/conda-forge/conda-forge-pinning) via [`conda-forge/conda-forge-pinning-feedstock`](https://github.com/conda-forge/conda-forge-pinning-feedstock)
+- 🔒 Has access to Azure, Anaconda.org (cf-staging)
+
 Package-wide dependency pins are defined in [conda_build_config.yaml](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml) in the [conda-forge/conda-forge-pinning-feedstock](https://github.com/conda-forge/conda-forge-pinning-feedstock).
 
 For more information on conda-forge wide package pins, please refer to [Globally pinned packages](pinning_deps.md#globally-pinned-packages).
 
 Please open a [PR](../glossary.md#term-PR) and/or an issue there, if you think a pin needs to be advanced. For more information on updating globally pinned packages, please refer to [Updating package pins](pinning_deps.md#update-pins).
 
-<a id="documentation"></a>
+### conda-forge-repodata-patches
+
+This repository creates the `repodata.json` patches used by the Anaconda.org to amend the metadata coming from the published packages.
+
+- ⚙️ Deployed in [Anaconda.org](https://anaconda.org/conda-forge/conda-forge-repodata-patches) via [`conda-forge/conda-forge-repodata-patches-feedstock`](https://github.com/conda-forge/conda-forge-repodata-patches-feedstock)
+- 🔒 Has access to Azure, Anaconda.org (cf-staging)
+
+### conda-forge-ci-setup
+
+This special feedstock provides a package that defines the logic to install and configure a common CI setup across providers.
+
+- ⚙️ Deployed in [Anaconda.org](https://anaconda.org/conda-forge/conda-forge-ci-setup) via [`conda-forge/conda-forge-ci-setup-feedstock`](https://github.com/conda-forge/conda-forge-ci-setup-feedstock)
+- 🔒 Has access to Azure, Anaconda.org (cf-staging)
 
 ### Documentation
```

Comment:
Should we rename this to "Website" and leave an anchor with `#documentation` right on top?

Reply:
Now that things are consolidating, I want to move the repos listed below under "Automatic maintenance" up here, reorder and categorize all the repos so that the more important ones (staged-recipes, documentation) are those most easily found by users. That may bring some changes in headings; let's defer discussions about final names until then, ok?

Reply:
Done. I did rename this to Website, so I'm resolving this conversation. Please comment again if you would like further changes.